views/dashboard/ehr.html.twig

```diff
@@ -111,8 +106,8 @@ function renderMetricsEhrPlots() {
                             fixedrange: true
                         },
                         xaxis: {
-                            fixedrange: false,
-                             tickformat: '%Y-%m-%d'
+                            fixedrange: true,
```

Comment:
This was leftover from trying to prevent the x-axis from converting the date to a timestamp, but inadvertently turned on the Plotly.js zoom tools, etc. Wasn't needed now that we're charting a single point in time.